db/migrate/20190514080151_add_reminder_flags_to_subscription.rb

```diff
@@ -0,0 +1,6 @@
+class AddReminderFlagsToSubscription < ActiveRecord::Migration[5.2]
```

Comment:
I think this will need its date made later because the working patterns changes had a more recent migration.